### DISCUSSION TITLE
能否使用chatgpt4.0的引擎

### DISCUSSION THREAD

**Discussion by liufumore** [timqian/openprompt.co#28] [General]
能否使用chatgpt4.0的引擎

**Comment by timqian** [comment]
我还在 waitlist, 如果可以了会第一时间支持

**Reply by robin12jbj** [reply]
期待中，感谢作者

**Comment by liufumore** [comment]
今天为什么不能用了？



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Tim Qian ***@***.***&gt;
发送时间: 2023年3月31日 23:37
收件人: timqian/openprompt.co ***@***.***&gt;
抄送: liufumore ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [timqian/openprompt.co] 能否使用chatgpt4.0的引擎 (Discussion #28)





 
我还在 waitlist, 如果可以了会第一时间支持

**Comment by timqian** [comment]
@liufumore 
我看了一下，应该是被墙了，如果你已经付款，但是没有科学上网的能力。可以发送付款邮箱给我，我给你退款
